content/project/team-leads.adoc

```diff
@@ -1,62 +1,115 @@
+* Onboarding new contributors to the teams they lead.
+* Mentorship and knowledge sharing within the team to ensure healthy officer rotation in the project,
+  with opportunity for team leads to move on to other roles or to take a break when needed.
+* Facilitating initiatives within the scope of the responsibility, maintaining the team's roadmap,
+  and ensuring that key team initiatives reflected on the link:/project/roadmap/[public Jenkins roadmap].
+* Representing the Jenkins project and community within the area of their responsibility.
+
+
+## Team Leads and Officers
+
+[[security]]
+### Security Officer
+
+Coordinates the security-related activities in the Jenkins project. This is an **elected role**. 
 
 * Run the Jenkins CERT meeting
 * link:https://wiki.jenkins.io/display/JENKINS/Rewards+for+reporting+security+issues[Manage sending gifts to qualifying reporters of resolved security issues]
 * Coordinate work on and releases of security fixes with plugin authors and Release Officer
 * Publish link:https://wiki.jenkins.io/display/JENKINS/Security+Advisorie[Security Advisories] (including CVE IDs and CVSS) and notify the link:https://groups.google.com/forum/#!forum/jenkinsci-advisories[mailing list]
 * Drive security policy definition/changes in the community
 * Represent the Jenkins project on security topics with third parties
+* Represent the https://www.jenkins.io/security/cna/[Jenkins CNA].
 
-## Infrastructure
+[[infrastructure]]
+### Infrastructure Officer
 
-Oversees the Jenkins project infrastructure.
+Oversees the Jenkins project infrastructure. This is an **elected role**.
 
-* Direct triage of issues
+* Direct triage of INFRA issues
 * Direct maintenance of Jenkins project infrastructure ('INFRA' issues)
 * Encourage contributors to INFRA project
-* Communicates maintenance windows and managing the execution of those windows
+* Communicate maintenance windows and manage execution of those windows
+* Coordinate Jenkins Infrastructure initiatives and define priorities, get key initiatives reflected on the link:/project/roadmap/[public Jenkins roadmap]
 
-## Events
+[[events]]
+### Events Officer
 
 _note: This team lead position was approved in the link:http://meetings.jenkins-ci.org/jenkins-meeting/2016/jenkins-meeting.2016-02-17-19.00.html[2015-02-17 governance meeting]_
 
-Has an active role in driving and managing Jenkins community events. Such events include Jenkins User Conference, Jenkins Area Meetups, other Jenkins community events (FOSDEM, SCALE), and speaking opportunities. 
+Has an active role in coordinating Jenkins community events and outreach programs.
+Such events include conferences where Jenkins is represented (cdCon, FOSDEM, SCALE, etc.),
+community-organized local and virtual meetups,
+contributor summits,
+outreach events (hackathons, Hacktoberfest, GSoC/GSoD, etc.), and speaking opportunities.
+This is an **elected role**. 
```

Comment:
For the Infrastructure Officer, the information that it is an elected role is right under the title and next to a simple description. I think it would be good to maintain that for all officers so that people don't have to search to determine if the role is elected or appointed.